src/pages/projects/build-modern-layouts-with-css-grid/index.tsx

```diff
@@ -0,0 +1,253 @@
+
+type LandingProps = {
+  course: any
+}
+
+const landingPage: FunctionComponent<LandingProps> = (props) => {
+  const {course} = props
+
+  const {
+    pricingPageFigmaUrl,
+    crmPageFigmaUrl,
+    pricingPageImageUrl,
+    crmPageImageUrl,
+    ogImage,
+  } = course?.projects
+
+  const introduction: any = find(course?.projects?.content, {
+    label: 'Introduction',
+  })
+  console.log('introduction', {introduction})
+  const challenges: any = find(course?.projects?.content, {label: 'Challenges'})
+  const pricingPageDescription: any = find(course?.projects?.content, {
+    label: 'Pricing Page description',
+  })
+  const crmPageDescription: any = find(course?.projects?.content, {
+    label: 'CRM page description',
+  })
+  const outcomes: any = find(course?.projects?.content, {label: 'Outcomes'})
+  const challengeResource: any = find(course?.projects?.content, {
+    label: 'Resource',
+  })
+  const pricingPageChallenge: any = find(course?.projects?.content, {
+    label: 'Pricing page challenge',
+  })
+  const dashboardPageChallenge: any = find(course?.projects?.content, {
+    label: 'Dashboard page challenge',
+  })
+  const studyQuestions: any = find(course?.projects?.content, {
+    label: 'Self-Study Questions',
+  })
+  const bestPractices: any = find(course?.projects?.content, {
+    label: 'Best Practices',
+  })
+  const Submission: any = find(course?.projects?.content, {label: 'Submission'})
+
+  const title = course.title
+
+  return (
+    <>
+      <NextSeo
+        description={course.description}
+        title={title}
+        titleTemplate={'%s | egghead.io'}
+        twitter={{
+          site: `@eggheadio`,
+          cardType: 'summary_large_image',
+        }}
+        openGraph={{
+          title,
+          description: course.description,
+          site_name: 'egghead',
+          images: [
+            {
+              url: ogImage,
+            },
+          ],
+        }}
+      />
+      <div className="bg-gray-50 dark:bg-gray-900 sm:-my-5 -my-3 -mx-5 p-5">
+        <div className="mb-10 pb-10 xl:px-0 px-5 max-w-screen-xl mx-auto">
+          <div className="mt-10 mb-16 text-center">
+            <div className="mb-10">
+              <Image priority src={course.image} height="270" width="270" />
+            </div>
+            <p className="text-lg md:text-2xl leading-6 text-gray-500">
+              Portfolio Project
+            </p>
+            <h1 className="text-2xl md:text-4xl font-bold mt-2">
+              {course.projects.title}
+            </h1>
+          </div>
+          <div className="flex flex-col justify-center items-start mx-auto max-w-screen-md mb-16 w-full px-4">
+            <main className="prose dark:prose-dark prose-lg max-w-none w-full">
+              <Markdown>{introduction.text}</Markdown>
+              <Markdown>{challenges.text}</Markdown>
+
+              <section className="relative dark:text-gray-200 mt-16 mb-16 ml-0 mr-0 md:-ml-20 md:-mr-20">
```

Comment:
added the challenges section to display the 2 different layouts side by side with description text